src/banks/registries/directory.py

```diff
@@ -38,24 +44,69 @@ def _load(self):
     def _scan(self):
         self._index: PromptFileIndex = PromptFileIndex()
         for path in self._path.glob("*.jinja*"):
-            pf = PromptFile(name=path.stem, version="0", path=path)
+            name, version = path.stem.rsplit(".", 1) if "." in path.stem else (path.stem, DEFAULT_VERSION)
+            pf = PromptFile(name=name, version=version, path=path, meta={})
             self._index.files.append(pf)
         self._index_path.write_text(self._index.model_dump_json())
 
-    def get(self, *, name: str, version: str | None = None) -> "Prompt":
-        version = version or "0"
+    def get_prompt(self, *, name: str, version: str | None = None) -> "Prompt":
```

Comment:
Why changing the method name? Its counterpart is still `set`

Reply:
Because with `set` we can set both meta and prompt. I thought being explicit might be better?

Reply:
Ah now I remember the convo, yeah let's scratch this. But I think `get_meta` will end up being just an utility method calling `get_prompt` under the hood? See comment below.